lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -85,82 +91,369 @@ describe('NxTable', function() {
       </NxTable>
     );
 
-    expect(component.find('td')).toHaveProp('colSpan', 2);
-    expect(component.find('td')).toHaveClassName('nx-cell--meta-info');
+    expect(screen.getByRole('cell')).toHaveAttribute('colSpan', '2');
+    expect(screen.getByRole('cell')).toHaveClass('nx-cell--meta-info');
   });
 
   it('Puts the correct colSpan on an emptyMessage meta-info cell', async function() {
-    let component: ReactWrapper;
-    await act(async () => {
-      component = mount(
-        <NxTable>
-          <NxTableHead>
-            <NxTableRow>
-              <NxTableCell>Foo</NxTableCell>
-              <NxTableCell>Bar</NxTableCell>
-            </NxTableRow>
-          </NxTableHead>
-          <NxTableBody emptyMessage="Empty" />
-        </NxTable>
-      );
-    });
-    component!.update();
+    render(
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <NxTableCell>Foo</NxTableCell>
+            <NxTableCell>Bar</NxTableCell>
+          </NxTableRow>
+        </NxTableHead>
+        <NxTableBody emptyMessage="Empty" />
+      </NxTable>
+    );
 
-    expect(component!.find('td')).toHaveProp('colSpan', 2);
-    expect(component!.find('td')).toHaveClassName('nx-cell--meta-info');
-    expect(component!.find('td')).toHaveText('Empty');
+    expect(screen.getByRole('cell')).toHaveAttribute('colSpan', '2');
+    expect(screen.getByRole('cell')).toHaveClass('nx-cell--meta-info');
+    expect(screen.getByRole('cell')).toHaveTextContent('Empty');
   });
 
   it('Puts the correct colSpan on an error meta-info cell', function() {
-    const component = mount(
+    render(
       <NxTable>
         <NxTableHead>
           <NxTableRow>
             <NxTableCell>Foo</NxTableCell>
             <NxTableCell>Bar</NxTableCell>
           </NxTableRow>
         </NxTableHead>
-        <NxTableBody error="Error!"/>
+        <NxTableBody error="Error!" />
       </NxTable>
     );
 
-    expect(component.find('td')).toHaveProp('colSpan', 2);
-    expect(component.find('td')).toHaveClassName('nx-cell--meta-info');
-    expect(component.find('td')).toIncludeText('Error!');
+    expect(screen.getByRole('cell')).toHaveAttribute('colSpan', '2');
```

Comment:
In JSX, `colSpan` uses camelCase, but isn't it usually entirely lowercase in the DOM?

Reply:
> In JSX, `colSpan` uses camelCase, but isn't it usually entirely lowercase in the DOM?

Oo for some reason it still passed so it looks like its not toHaveAttribute is not case sensitive?
I fixed that here: 90866a5d21352d7d363052acefeb0c54a20fb990